src/server/table_definitions.sql

```diff
@@ -0,0 +1,95 @@
+-- a minimal users table, if not using a phpbb3 installation
+-- DROP TABLE IF EXISTS users;
+-- CREATE TABLE users
+-- (
+--     user_id int(10) unsigned NOT NULL AUTO_INCREMENT,
+--     user_type tinyint(2) NOT NULL DEFAULT '0',
+--     username varchar(255) COLLATE utf8_bin NOT NULL DEFAULT '',
+--     user_password varchar(255) COLLATE utf8_bin NOT NULL DEFAULT '',
+--     user_email varchar(100) COLLATE utf8_bin NOT NULL DEFAULT '',
+--     PRIMARY KEY (user_id),
+--     KEY user_type (user_type)
+-- ) ENGINE=InnoDB AUTO_INCREMENT=50 DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
+
+------
+
+-- table used to manage bans issued by wesnothd
```

Comment:
The table is for forum bans and is only read from wesnothd. Bans issued from wesnothd go to `ban_save_file`